src/main/java/eu/europa/ted/eforms/sdk/util/XmlNoticeParser.java

```diff
@@ -0,0 +1,69 @@
+package eu.europa.ted.eforms.sdk.util;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Iterator;
+
+import javax.xml.XMLConstants;
+import javax.xml.namespace.NamespaceContext;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.xpath.XPath;
+import javax.xml.xpath.XPathExpressionException;
+import javax.xml.xpath.XPathFactory;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.w3c.dom.Document;
+import org.xml.sax.SAXException;
+
+import eu.europa.ted.eforms.sdk.domain.XmlNotice;
+
+public class XmlNoticeParser {
```

Comment:
Could we please use a uniform way of loading XML files?
It should either be the existing way implemented on the "XmlParser" class (which uses JAXB POJOs) or this one here on XmlNoticeParser.

Reply:
Renamed the class and added a class comment to explain the different purpose than XmlParser.